src/rules/effects/prefer-concat-latest-from.ts

```diff
@@ -21,43 +30,83 @@ export default createRule<Options, MessageIds>({
     version: '>=12.0.0',
     docs: {
       category: 'Possible Errors',
-      description:
-        'Use `concatLatestFrom` instead of `withLatestFrom` to prevent the selector from firing until the correct `Action` is dispatched.',
+      description: `Use \`${concatLatestFromKeyword}\` instead of \`${withLatestFromKeyword}\` to prevent the selector from firing until the correct \`Action\` is dispatched.`,
       recommended: 'warn',
-      suggestion: true,
     },
-    schema: [],
+    fixable: 'code',
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          strict: {
+            type: 'boolean',
+            default: defaultOptions.strict,
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
     messages: {
-      [messageId]: 'Use `concatLatestFrom` instead of `withLatestFrom`.',
-      [messageIdSuggest]: 'Replace `withLatestFrom` with `concatLatestFrom`.',
+      [messageId]: `Use \`${concatLatestFromKeyword}\` instead of \`${withLatestFromKeyword}\`.`,
     },
   },
-  defaultOptions: [],
-  create: (context) => {
+  defaultOptions: [defaultOptions],
+  create: (context, [{ strict }]) => {
+    const actionsName = findNgRxEffectActionsName(context)
```

Comment:
Do you see any problem in getting the `Actions` to report only in `Effects` that deal with actions? It was the cleanest way I could think of.